lib/generators/clearance/install/install_generator.rb

```diff
@@ -102,10 +102,16 @@ def migration_name_without_timestamp(file)
       end
 
       def users_table_exists?
-        if ActiveRecord::Base.connection.respond_to?(:data_source_exists?)
-          ActiveRecord::Base.connection.data_source_exists?(:users)
+        begin
+          connection = ActiveRecord::Base.connection
```

Comment:
I think extracting a `connection` method that returns this or raises a descriptive error on `NoDatabaseError` would be enough here.

In fact, even that may prove difficult because in rails 5 and newer the suggested command is `rails db:create` whereas in earlier version `rake db:create` should be run.